ui/pages/send/gas-display/gas-display.js

```diff
@@ -316,74 +319,78 @@ export default function GasDisplay({ gasError }) {
           ]}
         />
       </Box>
-      {(gasError || isInsufficientTokenError) && currentNetworkName && (
-        <Box
-          className="gas-display__warning-message"
-          data-testid="gas-warning-message"
-        >
+      {(gasError ||
+        isInsufficientTokenError ||
+        draftTransaction.asset.balance === '0x0' ||
+        draftTransaction.asset.balance === '0x00') &&
```

Comment:
If the use case for '0x00' is valid, then we should probably do something like 

```
const assetBalanceIsZero = draftTransaction.asset.balance === '0x0' || 
  draftTransaction.asset.balance === '0x00';
```

and then use that constant in the two places we need it.

Reply:
Okay I will do that